src/containers/BooksForm.js

```diff
@@ -10,31 +14,77 @@ const category = [
   { id: 7, name: 'Sci-fi' },
 ];
 
-const BooksForm = () => (
-  <div>
-    <form>
-      <label htmlFor="title">
-        The Title of the Book
-        <input
-          type="text"
-          placeholder="enter the title of the book"
-          name="title"
-          id="title"
-        />
-      </label>
-      <label htmlFor="select">
-        <select>
-          {category.map(c => (
-            <option key={c.id}>{c.name}</option>
-          ))}
-        </select>
-      </label>
-      <button type="submit" value="submit">
-        Submit
-      </button>
-    </form>
-  </div>
-);
-
-
-export default BooksForm;
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // id: 0,
```

Comment:
please delete commented codes 
![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png)